guix-systole/systoleos/systoleos.scm

```diff
@@ -0,0 +1,188 @@
+    (firmware (list linux-firmware iucode-tool amd-microcode))
+
+    (host-name "systole")
+    (timezone "Europe/Oslo")
+
+    ;; Use the UEFI variant of GRUB with the EFI System
+    ;; Partition mounted on /boot/efi.
+    (bootloader (bootloader-configuration
+                  (bootloader grub-efi-bootloader)
+                  (targets '("/boot/efi"))))
+
+    ;; Assume the target root file system is labelled "my-root",
+    ;; and the EFI System Partition has UUID 1234-ABCD.
+    (file-systems (append (list (file-system
+                                  (device (file-system-label "my-root"))
+                                  (mount-point "/")
+                                  (type "ext4"))
+                                (file-system
+                                  (device (uuid "1234-ABCD"
+                                                'fat))
+                                  (mount-point "/boot/efi")
+                                  (type "vfat"))) %base-file-systems))
+
+    ;; The account must be initialised with `passwd` command
+    (users (cons (user-account
+                   (name "systole")
+                   (comment "SystoleOS user")
+                   (password #f)
+                   (group "users")
+                   (supplementary-groups (list "wheel" "netdev" "audio"
+                                               "video"))) %base-user-accounts))
+
+    (packages (append (list
+                       ;; Slicer
+                       slicer-5.8
+
+                       ;; terminal emulator
+                       xterm
+
+                       ;; utils
+                       git
+                       curl
+                       vim
+                       stow) %base-packages))
+
+    (services
+     (append (list
+
+                   ;; LightDM display manager
+                   ;; Configuration documentation: https://guix.gnu.org/manual/en/html_node/X-Window.html
+                   (service lightdm-service-type
+                            (lightdm-configuration (allow-empty-passwords? #t)
+                                                   (debug? #t)
+                                                   (xdmcp? #t)
+                                                   (vnc-server? #f)
+                                                   (greeters (list (lightdm-gtk-greeter-configuration
+                                                                    (allow-debugging?
+                                                                     #t))))
+                                                   (seats (list (lightdm-seat-configuration
+                                                                 (name "*")
+                                                                 (user-session
+                                                                  "xfce.desktop"))))))
+
+                   ;; Services for xfce desktop environment
+                   (service xfce-desktop-service-type)
```

Comment:
@oeramo, XFCE may be overkilling for this purpose. in the `guix-psy-dicom` project I did use `fluxbox` (https://github.com/OUH-MESHLab/guix-psy-dicom/tree/enhancement/psydicom_system)

Reply:
@oeramo I would like to see autologin for the `brainlabmirror` user